scm/src/CMakeLists.txt

```diff
@@ -299,12 +299,16 @@ ADD_DEPENDENCIES(ccppphys ccpp)
 
 SET(scm_source_files gmtb_scm.F90
             gmtb_scm_input.F90
-            gmtb_scm_utils.F90
+            #gmtb_scm_utils.F90               #gets compiled with CCPP
```

Comment:
I was confused at first by "gets compiled with CCPP", of course it means that this is a dependency in CCPP. I would just remove those few lines altogether.

Reply:
addressed in last commit